src/pp.js

```diff
@@ -131,16 +131,26 @@ function hasHardLine(doc) {
   });
 }
 
-function _makeIndent(n) {
+function _makeIndent(tabWidth) {
   var s = "";
 
-  for (var i = 0; i < n; i++) {
+  for (var i = 0; i < tabWidth; i++) {
     s += " ";
   }
 
   return s;
 }
 
+function _repeatIndent(n, indent) {
```

Comment:
I want this to run in most browsers though, too, for demos and such. Is that function implemented in all modern browsers?